homeassistant/components/justnimbus/__init__.py

```diff
@@ -3,14 +3,18 @@
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
 
 from .const import DOMAIN, PLATFORMS
 from .coordinator import JustNimbusCoordinator
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up JustNimbus from a config entry."""
-    coordinator = JustNimbusCoordinator(hass=hass, entry=entry)
+    try:
+        coordinator = JustNimbusCoordinator(hass=hass, entry=entry)
+    except KeyError as err:
```

Comment:
I'd explicitly check the config entry data for the new key so we don't need to catch a general exception like KeyError here.

Reply:
Yeah, you're right, this is a bit nicer than catching a general exception. Thx for the great feedback and suggestions! I have updated the PR!